lab-brae/gulpfile.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const gulp = require('gulp');
+const eslint = require('gulp-eslint');
+const mocha = require('gulp-mocha');
+
+gulp.task('test', function() {
+  gulp.src('./test/*-test.js', { read: false })
+  .pipe(mocha({ reporter: 'spec' }));
```

Comment:
Try some of the other reporters if you get a chance. (Spec is boring, despite what Brian says. :p).

Try 'landing' or 'nyan'!